src/views/Correction.vue

```diff
@@ -127,13 +126,6 @@ export default class Correction extends Mixins(CommonMixin) {
         throw new Error('Invalid correction type')
       }
 
-      // do not proceed if not authorized
-      if (!this.isAuthorized) {
-        window.alert('You are not authorized to use Correction filings.')
-        this.$root.$emit('go-to-dashboard', true)
-        return
-      }
-
```

Comment:
This code block had to be down here -- after we determined from the draft what entity type this is for.

We no longer care about entity type, so I moved this block much higher in this function (ie, fail early).